### PR TITLE
Docker cleanup

### PR DESCRIPTION
**Pull request by BOHEUS**
Related to #7648 

@charlesBochet I've deleted only dev related commands in Makefile, if it's wrong, I'll change it ASAP

### CODE REVIEW

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 1]
```diff
@@ -261,9 +278,3 @@ This should work out of the box with the eslint extension installed. If this doe
 
 }
 ```
-
-#### Docker container build
-
-To successfully build Docker images, ensure that your system has a minimum of 2GB of memory available. For users of Docker Desktop, please verify that you've allocated sufficient resources to Docker within the application's settings.
-
-<ArticleEditContent></ArticleEditContent>
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/local-setup.mdx:269]: We should leave <ArticleEditContent></ArticleEditContent>

File: packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx [hunk 2]
```diff
@@ -4,6 +4,10 @@ icon: TbBrandDocker
 image: /images/user-guide/objects/objects.png
 ---
 
+<ArticleWarning>
+Docker containers are for production or self- hosting, for the contribution please check the [Local Setup](https://twenty.com/developers/local-setup).
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx:8]: self-hosting

**Review by charlesBochet — APPROVED** [review]
Thank you, I have left comments

**Comment by charlesBochet** [comment]
/oss.gg 300

File: packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx:8]: +1 @BOHEUS

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 3]
```diff
@@ -145,16 +145,33 @@ Twenty requires a redis cache to provide the best performances
 
 <ArticleTabs label1="Linux" label2="Mac OS" label3="Windows (WSL)">
   <ArticleTab>
+    <b>Option 1:</b> To provision your database locally:
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/local-setup.mdx:148]: your redis

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 4]
```diff
@@ -145,16 +145,33 @@ Twenty requires a redis cache to provide the best performances
...
     Use the following link to install Redis on your Linux machine: [Redis Installation](https://redis.io/docs/latest/operate/oss_and_stack/install/install-redis/install-redis-on-linux/)
+
+    <b>Option 2:</b> If you have docker installed:
+    ```bash
+    docker run -d --name my-redis-stack -p 6379:6379 redis/redis-stack-server:latest
+    ```
   </ArticleTab>
   <ArticleTab>
-    To provision your database locally with `brew`:
+    <b>Option 1:</b>To provision your database locally with `brew`:
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/local-setup.mdx:157]: your redis

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 5]
```diff
@@ -145,16 +145,33 @@ Twenty requires a redis cache to provide the best performances
...
     ```bash
     brew install redis
     ```
+
+    <b>Option 2:</b> If you have docker installed:
+    ```bash
+    make postgres-on-docker
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-website/src/content/developers/local-setup.mdx:164]: agree with greptile

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 6]
```diff
@@ -145,16 +145,33 @@ Twenty requires a redis cache to provide the best performances
...
+    ```
   </ArticleTab>
   <ArticleTab>
+    <b>Option 1:</b> To provision your database locally:
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/local-setup.mdx:168]: redis

**Comment by BOHEUS** [comment]
On it

File: packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx:8]: @charlesBochet in the same file there's absolute URL which I used as reference, update this URL to relative too?
image

[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-website/src/content/developers/self-hosting/docker-compose.mdx:8]: Ok, let's leave it as it is then! Consistence is more important

File: packages/twenty-website/src/content/developers/local-setup.mdx [hunk 7]
```diff
@@ -262,8 +279,6 @@ This should work out of the box with the eslint extension installed. If this doe
 }
 ```
 
-#### Docker container build
-
 To successfully build Docker images, ensure that your system has a minimum of 2GB of memory available. For users of Docker Desktop, please verify that you've allocated sufficient resources to Docker within the application's settings.
```
Comment (charlesBochet) [inline packages/twenty-website/src/content/developers/local-setup.mdx:282]: we should still remove this paragraph

Comment (charlesBochet) [reply packages/twenty-website/src/content/developers/local-setup.mdx:282]: @BOHEUS

Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/local-setup.mdx:282]: @charlesBochet but I've removed it earlier and restored as per this [comment](https://github.com/twentyhq/twenty/pull/7655#discussion_r1798472220), so... delete it or not? 😂

Comment (charlesBochet) [reply packages/twenty-website/src/content/developers/local-setup.mdx:282]: Delete the title + paragraph and keep the <ArticleEditContent></ArticleEditContent> :p

Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/local-setup.mdx]: Done!

**Pull request merged**